### PR TITLE
Add info icon with tooltip to test card device dropdown

### LINKED ISSUE #3088: Add info icon next to "Device" on the test card
### WHY?

Our assumption is that there are a lot of users who aren't aware of what device and/or swab type they're using. In an effort to educate users on the importance of device and swab type data accuracy and to get them to look into what device/swab type they're using, we're proposing to add an interactive info icon next to device. That way users can learn more about devices and what they can do to ensure accurate data submission.

**AS A** person conducting the test

**I WANT** click on an info icon next to "Device"

**SO THAT** I understand what action I can take to ensure accurate device and swab type data submission and why.

---

**GIVEN** an info icon next to the "Device" header

**WHEN** I click on it

**THEN** a tool tip with the following copy should pop up.

"Don’t see the test you’re using? Ask your organization admin to add the correct test and it'll show up here."

### DESIGN

Design notes:
- The icon is from font awesome
- The tool tip is an available component from USWDS (https://designsystem.digital.gov/components/tooltip/)

Figma link:
https://www.figma.com/file/NPSPM5SBzQ7Ra3mmr9M9cB/SimpleReport-Public-Site?node-id=5984%3A23809

### Translations
- [ ] Translations are required
- [ ] Translation request has been submitted
- [ ] Translations have been received

### PR DESCRIPTION
**Pull request by nathancrtr**
## Related Issue or Background Info
- Closes #3088 

## Changes Proposed
- Using the existing `TextWithTooltip` component, adds an info icon with tooltip to the device dropdown on a test card
 
## Additional Information
- As mentioned above, we already had a `TextWithTooltip` component that was well-suited to this use case
    - However, this component was only used by a single component: namely, the organization sign-up form
    - Owing to some deep CSS weirdness, the tooltip component had to be manually positioned (through `!important` negative margins) on the organization form in order to appear correctly
    - This position override did _not_ work for the test card, however
    - Therefore, I made some changes to the stylesheet so that the tooltip can appear in the correct position across different containers
    - This has the side effect of slightly changing the appearance of the tooltips as they appear on the sign-up form (for the better, imo) 
    - @vz3 check out the screenshots in my comment below and let me know if this 1) makes sense and 2) looks OK

## Screenshots / Demos
https://user-images.githubusercontent.com/27730981/156462871-73a20fdf-2372-41b5-9a45-36617d40266e.mov

## Checklist for Author and Reviewer
- @kenieh it looks like you did the designs for this work? Let me know if everything is to spec! 

### Infrastructure
- [x] **Consult the results of the `terraform-plan` job inside the "Terraform Checks" workflow run for this PR. Confirm that there are no unexpected changes!**

### Design
- [ ] Any UI/UX changes have a designer as a reviewer, and changes have been approved
- [ ] Any large-scale changes have been deployed to `test`, `dev`, or `pentest` and smoke-tested by both the engineering and design teams

### Content
- [x] Any content changes (including new error messages) have been approved by content team
    - See: https://app.zenhub.com/workspaces/simplereport-better-data-team-605b43d51ea9f700119a48ee/issues/cdcgov/prime-simplereport/3088#issuecomment-990523075 

### Support
- [x] Any changes that might generate new support requests have been flagged to the support team
- [x] Any changes to support infrastructure have been demo'd to support team

### Testing
- [x] Includes a summary of what a code reviewer should verify

### Changes are Backwards Compatible
- [x] Database changes are submitted as a separate PR
  - [x] Any new tables that do not contain PII are accompanied by a GRANT SELECT to the no-PHI user
  - [x] Any changes to tables that have custom no-PHI views are accompanied by changes to those views
        (including re-granting permission to the no-PHI user if need be)
  - [x] Liquibase rollback has been tested locally using `./gradlew liquibaseRollbackSQL` or `liquibaseRollback`
  - [x] Each new changeset has a corresponding [tag](https://docs.liquibase.com/change-types/community/tag-database.html)
- [x] GraphQL schema changes are backward compatible with older version of the front-end

### Security
- [x] Changes with security implications have been approved by a security engineer (changes to  authentication, encryption, handling of PII, etc.)
- [x] Any dependencies introduced have been vetted and discussed

## Cloud
- [x] DevOps team has been notified if PR requires ops support
- [x] If there are changes that cannot be tested locally, this has been deployed to our Azure `test`, `dev`, or `pentest` environment for verification

### CODE REVIEW

**Comment by nathancrtr** [comment]
@vz3 / @kenieh as mentioned in the PR notes, my changes to the tooltip component result in a few small side-effects on the organization sign-up form (for the better, if you ask me 😉)

**BEFORE**
Screen Shot 2022-03-02 at 5 34 03 PM
* No maximum width/word wrap
* Tooltip appears **above** the label
* Info icon is black
* Icon is `xs` size

**AFTER**
Screen Shot 2022-03-02 at 5 34 21 PM
* Word wrap
* Tooltip appears to the **right** of the label
* Info icon is "SimpleReport blue"
* Icon is `1x` size

Let me know if this looks OK. If we want to keep the sign-up form visually unaltered, I can update the component with the relevant configurable values.

**Review by kenieh — APPROVED** [review]
Nice updates! LGTM

File: frontend/src/app/commonComponents/TextWithTooltip.tsx [hunk 1]
```diff
@@ -47,12 +46,7 @@ export const TextWithTooltip = ({
       wrapperclasses="usa-text-with-tooltip"
     >
       {text}
```
Comment (zdeveloper) [inline frontend/src/app/commonComponents/TextWithTooltip.tsx:48]: is there a chance of this evaluating to `undefined` when not supplied?

**Review by zdeveloper — APPROVED** [review]
LGTM! thank you!

**Review by nickclyde — APPROVED** [review]
Looks great!

**Pull request merged**